src/components/DisclaimerPopup.vue

```diff
@@ -0,0 +1,296 @@
+              <td>NASA</td>
+              <td>sea surface temperature, Secchi depth, chlorophyll</td>
+              <td><a
+                  href="https://science.nasa.gov/earth-science/earth-science-data/data-information-policy/">https://science.nasa.gov/earth-science/earth-science-data/data-information-policy/</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+        <ul>
+          <li>
+            <p>5.4 Deltares expressly disclaims any implied warranty or
+              merchantability, fitness for a particular purpose, or for failure of
+              performance and any and all liability for any loss or damage whatsoever
+              suffered as a result of User's use or reliance upon the Viewer.</p>
+          </li>
+          <li>
+            <p>5.5 Furthermore, Deltares does not guarantee that the Viewer is
+              free of viruses and work without errors and/or interruptions.</p>
+          </li>
+          <li>
+            <p>5.6 To the best of Deltares' knowledge and belief, the Viewer
+              does not infringe any third-party copyright or any other intellectual
+              property. Any responsibility or liability of Deltares for claims, costs,
+              expenses and/or consequential losses or damages resulting from or
+              arising out of any infringement of intellectual property rights is
+              excluded.</p>
+          </li>
+          <li>
+            <p>5.7 User shall indemnify, hold harmless and defend Deltares
+              against any action brought by a third party against Deltares and/or its
+              officers, directors, employees or agents to the extent that such a claim
+              is connected to the use of the Viewer by User and/or third parties at
+              whose disposal User has placed the Website and/or Viewer.</p>
+          </li>
+        </ul>
+        <p>Article 6: Duration</p>
+        <ul>
+          <li>
+            <p>6.1 This User Agreement is concluded for an unlimited period of
+              time.</p>
+          </li>
+        </ul>
+        <p>Article 7: Privacy</p>
+        <ul>
+          <li>
+            <p>7.1 Any and all of User's personal data that User may provide to
+              Deltares for purposes of his/her use of the Viewer, will be processed in
+              accordance with the General Data Protection Regulation (EU) 2016/679 and
+              the Netherlands GDPR implementation Law 'Uitvoeringswet Algemene
+              verordering gegevensbescherming' (stb 2018, 145). Our Privacy Statement
+              sets out when and why we collect the data, exactly what that data is,
+              and how it will be processed and, above all, protected. Please read the
+              statement carefully, available on this website:
+              <a href="https://wetten.overheid.nl/BWBR0040940/2021-07-01">https://wetten.overheid.nl/BWBR0040940/2021-07-01</a>.</p>
```

Comment:
Remove hyperlink (based on advice of Riemer who mentions that links to dutch pages are not really useful)

Reply:
Added that one manually as it wasnt a link in the docx
Will change it back